inspirehep/modules/workflows/tasks/upload.py

```diff
@@ -45,7 +46,7 @@ def _get_updated_record(obj):
             )
         else:
             pid_type = get_pid_type_from_schema(obj.data['$schema'])
-            updated_record_id = obj.extra_data['record_matches'][0]
+            updated_record_id = get_value(obj.extra_data, 'matches.approved')
```

Comment:
we've to double check if the same behaviour for ``exact_match`` and ``fuzzy_match``

Reply:
Break if the approved is None or not there (key not there).